docs/advanced_topics/images/custom_image_model.md

```diff
@@ -67,3 +67,16 @@ work as before but would need to be updated in order to see any new images.
 
 .. autofunction:: get_image_model_string
 ```
+
+
+## Overiding upload location
+
+By default, all the image files will be saved to the "images" folder. This behavior is defined by `get_upload_to` function in the class `AbstractImage` and `AbstractRendition`.
```

Comment:
This is not fully correct. The default behaviour for saving images is a bit different for Rendition. Have another read of the actual methods and try to sum up each.

Reply:
Here's something pretty good from ChatGPT with some tweaks. 

```
By default, the `get_upload_to` method in `AbstractRendition` generates a file path in the "original_images" folder. Ensuring ASCII characters and limiting length to prevent filesystem issues during uploads.
```

And for the AbstractImage. 

```
The default `get_upload_to` method in `AbstractImage` prepares a file path within the "images" folder by combining the folder name and the validated filename.
```

It may be worth adding these ad docstrings on the methods and referring to them this way in the docs. 

See how we do this in the Page reference. 


https://github.com/wagtail/wagtail/blob/8ac19fff712d85de0b7c3b7f2739f661e73b04d9/docs/reference/pages/model_reference.md?plain=1#L157-L165

Reply:
"original_images" is created by method in AbstractImage not AbstractRendition right? Will start making changes to the docs now.

Reply:
Have a read of this docs section - it explains more about images & renditions. https://docs.wagtail.org/en/stable/advanced_topics/images/renditions.html

And make sure you've read the actual AbstractImage/Rendition code a bit more to understand where these methods are located and fit into the code flow. 

Thanks. This is good progress.